### ISSUE TITLE
Концептуальная схема данных предметной области

### ISSUE THREAD

**Issue by shiiishov** [shiiishov/kursovay#5]

**Comment by shiiishov** [comment]
![image](https://github.com/shiiishov/kursovay/assets/56388548/8eaa8260-2227-435c-bd5e-e4bdebb817ba)
Не понимаю есть ли смысл добавлять в схему сущность "Акции и новости", "Сообщения" (внутренние сообщения, например, о законченной стирке) и состояние прачечного оборудования и т.д. (оно же по логике должно подавать сигнал о каких-то поломках или когда средства для стирки закончились)

**Comment by wldmr-key** [comment]
ой! А куда все оборудование пропало?

чтой-то твоя концептуальная схема не очень согласуется с требованиями к системе

**Comment by shiiishov** [comment]
![image](https://github.com/shiiishov/kursovay/assets/56388548/e5691663-cb0f-40b3-8ffc-b69a4190858a)
с целого пинка добавил сущность оборудование. Состояние буду оценивать либо складывая время отработанное каждой машинки, либо считать кол-во запусков машинки)

**Comment by shiiishov** [comment]
![image](https://github.com/shiiishov/kursovay/assets/56388548/73cef9ff-f309-4189-8fae-ac29d32f232b)
исправил, то о чем говорили в зуме и не только. Добавил имя для клиентов, чтобы пользователи имели возможность написать как к ним обращаться. Коментарий сделал в ед. ч. Т.к. в заказе может быть один коммент (или пожелания) от клиента.

**Comment by wldmr-key** [comment]
в вариантах цена * количество = стоимость.Так? Тогда один из трех атрибутов лишний

Ну и описание предметной области стоит подогнать под схему ))

**Comment by shiiishov** [comment]
да, уберу стоимость услуги. В предметную область добавлю про это всё информацию. 

Другой вопрос. Есть смысл добавлять сумму заказа в сущность заказы? Чтобы запоминать сколько стоил тот или иной заказ? Атрибут этот будет вычисляться как сумма доставки плюс стоимость услуг (цена за кг * количество).

**Comment by wldmr-key** [comment]
Смысла нет. Ты прав в своих рассуждениях
